orc8r/cloud/go/obsidian/swagger/v1/swagger.yml

```diff
@@ -8933,6 +8933,19 @@ definitions:
     required:
     - latency_ms
     type: object
+  imei:
+    description: IMEI value
+    properties:
+      snr:
+        example: "176148"
+        pattern: ^(\d{6})$
+        type: string
+        x-nullable: false
+      tac:
+        $ref: '#/definitions/tac'
```

Comment:
this is a new field, not the same as tracking area code. let's define it as a locally scoped one just as snr here... there is no reuse by other definitions...